share/github-backup-utils/ghe-backup-config

```diff
@@ -66,6 +66,32 @@ for f in "$GHE_BACKUP_CONFIG" "$GHE_BACKUP_ROOT/backup.config" \
   fi
 done
 
+ghe_parallel_check() {
+  if [ "$GHE_PARALLEL_ENABLED" != "yes" ]; then
+    return 0
+  fi
+
+  # Some machines may have both moreutils parallel and GNU parallel installed.
+  GHE_PARALLEL_COMMAND="parallel"
+  if [ -x "/usr/bin/parallel.moreutils" ]; then
+    GHE_PARALLEL_COMMAND="/usr/bin/parallel.moreutils"
+  fi
+
+  # Check that the GHE_PARALLEL_COMMAND is pointing to moreutils parallel
+  if ! $GHE_PARALLEL_COMMAND -h | grep -q "parallel \[OPTIONS\] command -- arguments"; then
+    echo "Error: moreutils not found. Please install https://joeyh.name/code/moreutils" 1>&2
+    exit 1
+  fi
+
+  if [ -n "$GHE_PARALLEL_MAX_JOBS" ]; then
+    GHE_PARALLEL_COMMAND_OPTIONS="-j $GHE_PARALLEL_MAX_JOBS"
```

Comment:
Not sure we need this space, but the option just below has it -- I'm looking to see where they're used to check but wanted to note it in the mean time...

```suggestion
    GHE_PARALLEL_COMMAND_OPTIONS=" -j $GHE_PARALLEL_MAX_JOBS"
```

Reply:
Yep this was intentional, thanks.